index.js

```diff
@@ -1,9 +1,9 @@
-// Main module entry point for nodereport
+// Main module entry point for node-report
 
 const api = require('./api');
 
-// NODEREPORT_EVENTS env var overrides the defaults
-const options = process.env.NODEREPORT_EVENTS || 'exception+fatalerror+signal+apicall';
+// node-report_EVENTS env var overrides the defaults
+const options = process.env.node-report_EVENTS || 'exception+fatalerror+signal+apicall';
```

Comment:
We want to keep the environment variables case (upper), probably without the hyphen.